files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md

```diff
@@ -1,57 +1,90 @@
 ---
-title: Function.arguments
+title: Function.prototype.arguments
 slug: Web/JavaScript/Reference/Global_Objects/Function/arguments
 ---
 
-{{JSRef}} {{deprecated_header}}
+{{JSRef}}{{Deprecated_Header}}{{Non-standard_Header}}
 
-`function.arguments` 属性代表传入函数的实参，它是一个类数组对象。
+**备注：** {{jsxref("Function")}} 对象的 `arguments` 属性已被弃用。访问 `arguments` 对象的推荐方式是在函数内部引用可用的变量 {{jsxref("Functions/arguments", "arguments")}}。
+
+{{jsxref("Function")}} 实例的 **`arguments`** 访问器属性返回传递给该函数的参数。对于[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)、箭头函数、异步函数和生成器函数，访问 `arguments` 属性会抛出 {{jsxref("TypeError")}}。
 
 ## 描述
 
-`function.arguments` 已经被废弃了，现在推荐的做法是使用函数内部可用的 {{jsxref("Functions/arguments", "arguments")}} 对象来访问函数的实参。
+`arguments` 的值是一个类数组对象，对应于传递入函数的参数。
+
+在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。
+
+如果函数没有正在进行的未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。
+
+请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且实现不允许改变此语义，除非是非严格的普通函数。`arguments` 属性的实际行为如果不是抛出错误，就是由实现定义的。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 将初始的毒丸访问器 `Function.prototype.arguments` 扩展为特殊处理非严格函数的 `this` 值。
```

Comment:
I'm confused about the translation of these two paragraphs. Maybe we can improve them.